agent/xds/server_test.go

```diff
@@ -315,15 +315,15 @@ func expectEndpointsJSON(t *testing.T, snap *proxycfg.ConfigSnapshot, token stri
 	}`
 }
 
-func expectedUpstreamTLSContextJSON(t *testing.T, snap *proxycfg.ConfigSnapshot, sni string) string {
-	return expectedTLSContextJSON(t, snap, false, sni)
+func expectedUpstreamTLSContextJSON(snap *proxycfg.ConfigSnapshot, sni string) string {
+	return expectedTLSContextJSON(snap, false, sni)
 }
 
 func expectedPublicTLSContextJSON(t *testing.T, snap *proxycfg.ConfigSnapshot) string {
```

Comment:
I ended up adding an exclude rule for `t` because there were so many. A fixed a few before realizing that I was going to need to exclude `t`.  I think a good second pass would be to fix all the unused `t` functions.